ckan/public/base/javascript/modules/resource-view-filters.js

```diff
@@ -60,6 +60,18 @@ this.ckan.module('resource-view-filters', function (jQuery) {
   }
 
   function _appendDropdowns(dropdowns, resourceId, template, fields, filters) {
+    //TODO: handle decoding the filters object to be able to properly available in the current filter dropdowns
+    console.log(filters);
```

Comment:
Maybe remove this to avoid noise
```suggestion
    console.log(filters);
```